extensions/eclipse-che-theia-plugin-ext/src/resource/che-logo.svg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I see where is used the che-logo.svg, but che-logo.png ?

Reply:
PNG has been removed as unused. Thanks.